FND/GoalsAndObjectives/Objectives.rdf

```diff
@@ -118,7 +130,7 @@
 	<owl:Class rdf:about="&fibo-fnd-gao-obj;Strategy">
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-gao-obj;hasObjective"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-gao-obj;isDesignedToAchieve"/>
```

Comment:
What's the reason for this change? I'd prefer to get rid of the unnecessary property isDesignedToAchieve, which is only used in one place CIV. In contrast hasObjective is used in about 10 places. isDesignedToAchieve is a misnomer anyway: an objective could be imposed on something AFTER it was originally designed and running. e.g. a statistical (or other) program may have been running several years and this year the executive says "your objective this year is to collect 1000 samples".

Reply:
Ok - I've removed it, and made the restriction min 0, and added a min 0 restriction for hasGoal given the definition.  I've deleted the property however.